public/locales/pt/editor.json

```diff
@@ -0,0 +1,16 @@
+{
+	"editor-desktop-optimized": "Esse site é otimizado para o desktop",
```

Comment:
```suggestion
	"editor-desktop-optimized": "Este site está otimizado para o desktop",
```

Reply:
I think this translation is ok, don't make much difference IMO

Reply:
I know in pt-BR there "esse" and "este" can be used as synonyms, but for pt-PT it's the difference between "this" and "that" (the latter being wrong, of course). If it doesn't make a difference for pt-BR, please accept it then, since it does make a difference for pt-PT.

Reply:
Nice, didn't know that was a difference between `esse` and `este` in pt-PT, let's go with your suggestion